InternetHospital.BusinessLogic/Services/FeedBackService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using InternetHospital.BusinessLogic.Interfaces;
+using InternetHospital.BusinessLogic.Models;
+using InternetHospital.DataAccess;
+using InternetHospital.DataAccess.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace InternetHospital.BusinessLogic.Services
+{
+    public class FeedBackService : IFeedBackService
+    {
+
+        private readonly ApplicationContext _context;
+
+        public FeedBackService(ApplicationContext context)
+        {
+            _context = context;
+        }
+
+        public void FeedBackCreate(FeedBackCreationModel model, int userId)
```

Comment:
this method saves model to the database. It's not a good practive with void signature

Reply:
What type is better to use?

Reply:
used user class FeedBack